machines/vcsa/README.md

```diff
@@ -0,0 +1,79 @@
+# vCenter cluster testbed automation
+
+## Overview
+
+This directory contains scripts to automate VCSA/ESXi install and cluster configuration for developing and testing VIC.
+
+## Dependencies
+
+### govc
+
+Most of the heavy lifting is done by govc, install via:
+
+```
+go get -u github.com/vmware/govmomi/govc
+```
+
+### jq
+
+Used here to derive static VCSA networking from its parent ESXi host.
+But, you should already be using and loving jq for other tasks: http://stedolan.github.io/jq/
+
+```
+brew install jq
+```
+
+## Scripts
+
+### create-esxi-vm.sh
+
+This script creates a VM running stateless ESXi, booted via cdrom/iso.
+It will create 2 disks:
+
+* vSAN cache disk (Virtual SSD)
+
+* vSAN store disk
+
+The two vSAN disks will be unformatted, leaving them to be autoclaimed
+by a vSAN enabled cluster.
+
+Note that for a cluster to use vSAN, it will need at least 3 of these
+ESXi VMs.
+
+### create-vcsa-vm.sh
+
+This script creates a VM with VCSA (Virtual Center Server Appliance)
+installed.
+
+### create-cluster.sh
```

Comment:
What does cluster here mean specifically?  Does it mean create the VSAN datastore across the esxi instances and attach the esxi instances to the VC created above?  Or something else?